go/oasis-test-runner/oasis/validator.go

```diff
@@ -99,7 +101,7 @@ func (net *Network) NewValidator(cfg *ValidatorCfg) (*Validator, error) {
 		"registry", "node", "init",
 		"--datadir", val.dir.String(),
 		"--node.consensus_address", fmt.Sprintf("127.0.0.1:%d", val.consensusPort),
-		"--node.expiration", "1000000",
+		"--node.expiration", "1",
```

Comment:
At one point, this would break horribly if it wasn't at least "2", but that might have been fixed.

Reply:
Yes, @kostko, told me that it should be `1` or `2`.

I tried with `1` and the tests passed :slightly_smiling_face:.

Reply:
Validators expiring seems really dangerous though. I think the scheduler will currently even allow having only a single validator which could then do arbitrary stuff?

Reply:
> I think the scheduler will currently even allow having only a single validator which could then do arbitrary stuff?

Correct.  However:
 * We have not established the minimum number of validators that we want (and it is a policy decision that I am not qualified to make).  The correct thing to do is for someone to come up with validator committee minimum parameters, that we enforce.
 * Even with such parameters, the only safe behavior in the event that we do not have a sufficient node pool to elect a validator committee is to halt.

Reply:
Agreed, https://github.com/oasislabs/oasis-core/issues/2314.